### PR TITLE
Organize README and create CONTRIBUTING

### LINKED ISSUE #93: Request for documentation on building a docker image from source
Being that this is bank related it would be safer if I were to build a docker image from source rather than the released version. However, I have not used this tech stack before and I don't see a dockerfile. The HELP.md page linked to very generic information for building an image. Could anyone point me in the correct direction to building my own image and deploying? Im happy to help write documentation as well as soon as I understand it better

Thanks :)

### PR DESCRIPTION
**Pull request by CraftingGamerTom**
closes #93

### CODE REVIEW

**Comment by CraftingGamerTom** [comment]
@dvankley Would you review this and let me know if there is anything I should change to get this merged in?

I recommend using a VS Code Markdown viewer if hard to see formatting by eye

**Comment by dvankley** [comment]
Hey @CraftingGamerTom, thanks for the contribution.

While I don't think the README is perfect, I'm not sure it needs a wholesale reorganization and rewrite. Maybe that's just my ego talking.

Assuming I get past that hangup, there are a non-zero number of typos, grammatical issues, etc. in this change. That's not intended to be a criticism, but it means I need to go through and proofread/edit your change, in addition to checking for correctness. This is not unreasonable, but your change is substantial, and combined with my previous point about the necessity of this change the result is that I don't have a lot of motivation to actually do the work to get this merged in.

Bottom line: I will try to get around to this eventually, but it's not high on my priority list.

To give a more focused answer to your question if there's anything that you should change to get this merged in: I wouldn't say you _should_ do this, but if you really wanted to move this forward faster you _could_ try to reduce the size of the change to make it more digestible, as well as proofread it for grammar and spelling.

**Comment by CraftingGamerTom** [comment]
Ah thank you for the feedback! Sure thing - great, well handled response. I'll take another swing at it some time

**Comment by CraftingGamerTom** [comment]
Also, Im sorry for any offense I caused - your README certainly got many people successfully working with plaid/firefly. The rewrite helped me understand a bit more of the order - and personally struggled past some of the first hand accounts that I would start getting lost (probably my ADHD). 

I will try to cut up the PR and make the reasoning for the changes more clear :)

**Comment by dvankley** [comment]
@CraftingGamerTom no offense taken at all. I appreciate your contribution, and I appreciate you understanding why I wasn't able to merge it straight away. I'm sure we can work together on improving the documentation for everyone.
